src/app/features/courses/components/courses-edit/courses-edit.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="courses-edit">
+  <h1 class="courses-edit__title">{{ pageTitle }}</h1>
+
+  <app-input
+    label="Title"
+    type="text"
+    name="title"
+    placeholder="Text input"
+    [required]="true"
+    [(inputModel)]="course.title"
+  ></app-input>
+
+  <app-input
```

Comment:
In the task there is a requirement - do not use custom form controls, because it's quite complicated for you right now and requires certain deep knowledge on how to implement it. I encourage you to use plain input element in this form or if you really want to use your custom implementation, please, look into this [article](https://blog.angular-university.io/angular-custom-form-controls/).

Also, just for the future, try to use template-driven forms as rarely as possible. We have an amazing [Reactive Forms](https://angular.io/guide/reactive-forms) mechanism instead.  This is what you should use in real angular projects in 90% of all cases.

Reply:
fixed